cinema/views.py

```diff
@@ -5,54 +1,151 @@
+from django.db.models import F, Count
+
+from rest_framework import viewsets, status
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from cinema.models import (
+    Genre,
+    Actor,
+    CinemaHall,
+    Movie,
+    MovieSession,
+    Order
+)
 from cinema.serializers import (
     GenreSerializer,
     ActorSerializer,
     CinemaHallSerializer,
     MovieSerializer,
     MovieSessionSerializer,
-    MovieSessionListSerializer,
-    MovieDetailSerializer,
     MovieSessionDetailSerializer,
     MovieListSerializer,
+    OrderListSerializer,
+    OrderSerializer,
+    MovieGetDetailSerializer,
+    MovieSessionListSerializer,
 )
 
 
 class GenreViewSet(viewsets.ModelViewSet):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
 
+    def list(self, request, *args, **kwargs):
+        response = super().list(request, *args, **kwargs)
+        genres_data = response.data.get("results", response.data)
+        return Response(genres_data, status=status.HTTP_200_OK)
+
 
 class ActorViewSet(viewsets.ModelViewSet):
     queryset = Actor.objects.all()
     serializer_class = ActorSerializer
```

Comment:
The 'ActorViewSet' also uses a custom 'list' method similar to 'GenreViewSet'. Ensure that this approach is required and does not conflict with pagination or other response features.